exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,21 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Ken";
+var mentor5 = "Yohannes";
+
+function shout() {
+    return "HELLO ";
+};    

```

Comment:
I like how you have abstracted this little bit of logic into a function - you could also do this for the logic that makes the name upper case